app/models/owner.rb

```diff
@@ -18,7 +18,7 @@ class Owner < ActiveRecord::Base
   serialize :feature_switches
 
   # Supporters that are on a particular plan
-  scope :supporters, ->(plan) { where(stripe_plan_id: plan.stripe_plan_id) }
+  scope :supporters, ->(plan) { where(stripe_plan_id: plan.stripe_plan_id).order("RAND()") }
```

Comment:
This is effectively setting a default order of random on this scope. This feels like it could become very confusing because it's easy to miss - you'd be wondering why you were getting different results each time you call this. This is the same reason [default scopes are to be avoided](http://www.databasically.com/2011/02/07/default_scope-use-cases-caveats-and-work-arounds/).

For this reason I'd suggest doing this at the controller level. What do you think?